package-scripts.js

```diff
@@ -6,13 +6,10 @@ module.exports = {
     },
     test: {
       default: {
-        description: 'Runs AVA with nyc (which is configured in package.json)',
-        script: 'cross-env NODE_ENV=test nyc ava',
-      },
-      watch: {
-        description: 'Run AVA in watch mode',
-        script: 'ava -w --require babel-register',
+        description: `This runs jest with coverage. If we're on Travis, then we'll ignore the cache (just in case).`,
+        script: `jest --coverage ${process.env.CI ? '--no-cache' : ''}`,
```

Comment:
I'd considered that. Probably doesn't need to be there. But because I bumped into an issue which I think  may have been cache related I thought I'd make it explicit. Also I got excited at how easy this is with `p-s` :)